examples/ES/actor.py

```diff
@@ -0,0 +1,125 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gym
+import parl
+import time
+import numpy as np
+from es import ES
+from obs_filter import MeanStdFilter
+from mujoco_agent import MujocoAgent
+from mujoco_model import MujocoModel
+from noise import SharedNoiseTable
+
+
+@parl.remote_class(wait=False)
+class Actor(object):
+    def __init__(self, config):
+        self.config = config
+
+        self.env = gym.make(self.config['env_name'])
+        self.config['obs_dim'] = self.env.observation_space.shape[0]
+        self.config['act_dim'] = self.env.action_space.shape[0]
+
+        self.obs_filter = MeanStdFilter(self.config['obs_dim'])
+        self.noise = SharedNoiseTable(self.config['noise_size'])
+
+        model = MujocoModel(self.config['obs_dim'], self.config['act_dim'])
+        algorithm = ES(model)
+        self.agent = MujocoAgent(algorithm, self.config)
+
+    def _play_one_episode(self, add_noise=False):
+        episode_reward = 0
+        episode_step = 0
+
+        obs = self.env.reset()
+        while True:
+            if np.random.uniform() < self.config['filter_update_prob']:
+                obs = self.obs_filter(obs[None], update=True)
+            else:
+                obs = self.obs_filter(obs[None], update=False)
+
+            action = self.agent.predict(obs)
+            action = action.detach().numpy()
```

Comment:
This line should be inside the function `agent.predict`.

Reply:
fix